modules/apps/headless/headless-builder/headless-builder-api/src/main/java/com/liferay/headless/builder/constants/HeadlessBuilderConstants.java

```diff
@@ -19,4 +19,14 @@ public class HeadlessBuilderConstants {
 
 	public static final String BASE_PATH_SUFFIX = "/c/";
 
+	public static final String GROUP = "group";
```

Comment:
Maybe we could use the Scope enum instead, or including the value as in my previous comment with the retrieve type :thinking: